start-all.sh

```diff
@@ -5,7 +5,7 @@ VERSIONS=$DEFAULT_VERSION
 usage="$(basename "$0") [-h] [-d Prometheus data-dir] [-v comma seperated versions] [-g grafana port ] [ -p prometheus port ] -- starts Grafana and Prometheus Docker instances"
 
 GRAFANA_VERSION=4.1.1
-PROMETHEUS_VERSION=v1.5.2
+PROMETHEUS_VERSION=v1.4.1
```

Comment:
Did you test it? It did not works for me
Make sure to do a hard reload between test, otherwise you are using the cached JS